src/core/event.js

```diff
@@ -472,9 +481,11 @@ Candy.Core.Event = (function(self, Strophe, $) {
 					reason = reasonNode.text();
 				}
 
+				fromUser.jid = mediatedInvite.attr('from');
```

Comment:
In this case, because it's empty. It's being set above as a blank `new Candy.Core.Contact` and that data isn't set. Because there are two paths here, --oh. Hi. I meant to remove this line when I gave up on trying to consolidate it into one place before the if's and just put them in separately. Thanks. :)